images/win/scripts/Installers/Install-NodeLts.ps1

```diff
@@ -15,11 +15,9 @@ $queryFilter = "`$filter=(Id eq 'nodejs') and (IsPrerelease eq false) and (Versi
 $url = "https://community.chocolatey.org/api/v2/Packages()?$queryFilter"
 $chocoVersionsOutput = (Invoke-RestMethod -Uri $url).properties.Version
 $highestVersion = $chocoVersionsOutput `
-    | Where-Object { $_ -Like "$defaultVersion.*" } `
+    | Where-Object { $_ -Like "$defaultVersion.*" -or $_ -eq $defaultVersion } `
```

Comment:
```suggestion
    | Where-Object { $_ -match "^$defaultVersion" } `
```

Reply:
This works incorrectly for defaultVersion like "14.1" and returns "14.18.1" instead of "14.1.0"

Reply:
Do we ever need to install the `Major.Minor` version? In toolset we only have the Major one

Reply:
And even there such a case you can use match
`Where-Object { $_ -match "^$defaultVersion\." } `
![image](https://user-images.githubusercontent.com/48208649/140089230-bf9ce2a5-768f-480d-b4be-64f6b4cf31b4.png)

Reply:
`Where-Object { $_ -match "^$defaultVersion\." }` works incorrectly for the full version number like `14.18.1`. I would like to have flexible function which can return correct version for any default version, not only for Major.

Reply:
why do we need to set the full version in the toolset? The toolset is not intended for that

Reply:
Can you guarantee that at some moment we don't have to stick to a specific version due to compatibility issues for example? I guess that is a good option to be safe in future.